packages/block-library/src/buttons/edit.js

```diff
@@ -22,15 +22,14 @@ const alignmentHooksSetting = {
 
 function ButtonsEdit() {
 	const blockWrapperProps = useBlockWrapperProps();
+	const innerBlocksProps = useInnerBlocksProps( blockWrapperProps, {
```

Comment:
I wonder if you could do this in reverse, i.e. `const blockWrapperProps = useBlockWrapperProps( innerBlocksProps, { (etc.)`. I'm not saying you should, but I'm curious if it works either way.

Reply:
Sure, it could work, because we control both hooks and they don't have unmergeable props.

Reply:
The same will be possible for `useRichText`, you can stack it in any order.